book/doctrine.rst

```diff
@@ -337,8 +339,10 @@ in a number of different formats including YAML, XML or directly inside the
 
 .. tip::
 
-    The table name is optional and if omitted, will be determined automatically
-    based on the name of the entity class.
+    The table name annotation is optional. If it's omitted, Doctrine will
+    assume that the entity's class name should double as the database table
+    name. In the example above, an explicit definition was provided to force
+    the table name to be lowercased.
```

Comment:
I actually changed this back - it's a bit more complicated nowadays, because the SE comes with the lowercase strategy (https://github.com/symfony/symfony-standard/blob/master/app/config/config.yml#L59) so the table name does not necessarily equal the entity name.